src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
+    : people.filter((p) => p.sex === 'f');
```

Comment:
```suggestion
    ? people.filter(
      (p) => p.sex === "f" && people.find((p1) => p1.mother === p.name)
    )
    : people.filter((p) => p.sex === "f");
```

Try to format your code like that, so it will look prettier